x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/prebuilt_rules/prebuilt_rules_preview.cy.ts

```diff
@@ -1136,5 +1136,89 @@ describe('Detection rules, Prebuilt Rules Installation and Update workflow', ()
         cy.get(UPDATE_PREBUILT_RULE_PREVIEW).contains('execution_summary').should('not.exist');
       });
     });
+
+    describe(
+      'Viewing rule changes in per-field diff view',
+      {
+        tags: TEST_ENV_TAGS,
+        env: {
+          ftrConfig: {
+            kbnServerArgs: [
+              `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+                'perFieldPrebuiltRulesDiffingEnabled',
+              ])}`,
+            ],
+          },
+        },
+      },
+      () => {
+        it('User can see changes in a side-by-side per-field diff view', () => {
+          clickRuleUpdatesTab();
+
+          openRuleUpdatePreview(OUTDATED_RULE_1['security-rule'].name);
+          assertSelectedPreviewTab(PREVIEW_TABS.UPDATES); // Should be open by default
+
+          cy.get(UPDATE_PREBUILT_RULE_PREVIEW).contains('Current rule').should('be.visible');
+          cy.get(UPDATE_PREBUILT_RULE_PREVIEW).contains('Elastic update').should('be.visible');
+
+          cy.get(PER_FIELD_DIFF_WRAPPER).should('have.length', 2);
+
+          /* Version should be the first field in the order */
+          cy.get(PER_FIELD_DIFF_WRAPPER).first().contains('Version').should('be.visible');
+          cy.get(PER_FIELD_DIFF_WRAPPER).first().contains('1').should('be.visible');
+          cy.get(PER_FIELD_DIFF_WRAPPER).first().contains('2').should('be.visible');
+
+          cy.get(PER_FIELD_DIFF_WRAPPER).last().contains('Name').should('be.visible');
+          cy.get(PER_FIELD_DIFF_WRAPPER).last().contains('Outdated rule 1').should('be.visible');
+          cy.get(PER_FIELD_DIFF_WRAPPER).last().contains('Updated rule 1').should('be.visible');
+
+          /* Select another rule without closing the preview for the current rule */
```

Comment:
nit: Starting from this point it looks like a new scenario. It shouldn't be a significant overhead to move lines below to a separate scenario but will help us catch problems related to rules switching more efficiently. On top of that the test can varify `Outdated rule 2` content as well to make sure not only title changed.